tests/e2e/package.json

```diff
@@ -6,11 +6,11 @@
     "lint": "eslint src/*.js ./*.js"
   },
   "dependencies": {
-    "@chainsafe/dappeteer": "^2.4.0",
+    "@chainsafe/dappeteer": "^3.0.0",
     "ethereumjs-tx": "^2.1.2",
-    "jest-environment-node": "^28.1.3",
-    "puppeteer": "^17.0.0",
-    "web3": "^1.7.5",
+    "jest-environment-node": "^29.1.2",
```

Comment:
```suggestion
```

there will not need "jest-environment-node", new version jest make it as a buildin package.